pom.xml

```diff
@@ -1080,7 +1079,7 @@
       <dependency>
         <groupId>org.scalanlp</groupId>
         <artifactId>breeze_${scala.binary.version}</artifactId>
-        <version>2.0</version>
```

Comment:
please also add the upgrade of breeze in PR title and description

Reply:
okay